src/main/kotlin/com/hibob/academy/resource/petsResource.kt

```diff
@@ -17,22 +18,26 @@ class PetsResource {
     @Consumes(MediaType.APPLICATION_JSON)
     fun createPet(@RequestBody pets: Pets): Response {
         Response.status(Response.Status.OK).build()
-        return Response.ok("POST OK").build()
+        return Response.ok(pets).build()
     }
 
     @GET
     @Path("/{petId}/type")
     fun getPetType(@PathParam("petId") id: String): Response {
         Response.status(Response.Status.OK).build()
-        return Response.ok("GET OK").build()
+        return Response.ok(listOf(Pets(name = "dog", type = "lab"
+            , company_id = UUID.randomUUID(), date_of_arrival = Date() ))).build()
     }
 
     @PUT
-    @Path("/{petId}/type")
+    @Path("/{petId}")
```

Comment:
I think that the URL should indicate that we are updating only the type here, maybe it can be something like:
```suggestion
    @Path("/updateType/{petId}")
```